lib/proto.js

```diff
@@ -91,13 +92,31 @@ function child (bindings) {
       instance[serializersSym][bks] = bindings.serializers[bks]
     }
   } else instance[serializersSym] = serializers
+  if (bindings.hasOwnProperty('formatters')) {
+    instance[formattersSym] = Object.create(null)
+    for (k in formatters) {
```

Comment:
since formatters is supplied by user, we can't rely on for in without hasOwnProperty. 

formatters only has three properties, we should optimize for speed by just looking for those properties:

```
instance[formattersSym] = {
  level: formatters.level,
  bindings: formatters.bindings 
}
```

this is both less code, and faster

Reply:
Good point!